components/first_time/first_time_component.js

```diff
@@ -1,126 +1,75 @@
-import React, { Component } from 'react';
-import { Text, View, AppRegistry, StyleSheet, TextInput, ScrollView, Button, TouchableOpacity } from 'react-native';
+import React from 'react'
+import {Container, Header, Body, Title, Text, Form, Content, View, Button, Item, Input, Label} from 'native-base'
+import axios from 'axios'
 
 export default class FirstTimeComponent extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state={
-            field1: " ",
-            field2: " ",
-            field3: " ",
-            field4: " ",
-            field5: " "
-        };
+  constructor (props) {
+    super(props)
+    this.state = {
+      field1: '',
```

Comment:
I'd go ahead and make these more descriptive, so that it is easier to tell what's going on.